cspell.json

```diff
@@ -361,7 +361,12 @@
         "poap",
         "irss",
         "CELO",
-        "flac"
+        "flac",
```

Comment:
plz sync develop branch and run `gulp reorder-spellcheck` reorde cspell.json file